doc/time/zone.rdoc

```diff
@@ -0,0 +1,7 @@
+- +zone+: a timezone, which may be:
+  - A string offset from UTC.
+  - A single letter offset from UTC, in the range <tt>'A'..'Z'</tt>,
+    <tt>'J'</tt> (the so-called military timezone) excluded.
```

Comment:
I think it would be helpful to give examples and/or additional documentation for these two cases since it's not very clear how they work. For the first one, maybe give an example like `+09:00` for UTC+9. For the second one, it might be helpful to link to the [Wikipedia article](https://en.wikipedia.org/wiki/List_of_military_time_zones).